terraform/environments/electronic-monitoring-data/share_cloud_platform.tf

```
@@ -191,6 +191,24 @@ module "share_specials_data_marts" {
   role_arn                = module.specials_cmt_front_end_assumable_role.iam_role_arn
 }
 
+resource "aws_lakeformation_permissions" "ac_allied_db" {
+  count       = local.is-development || local.is-test ? 1 : 0
```

```suggestion
  count       =  local.is-test ? 1 : 0
```
this might be a recurring theme - are the dbs in dev??